config/vufind/channels.ini

```
@@ -13,7 +13,7 @@ default_home_source = "Solr"
 ;default_home_search = "id:my_favorite_record OR *:*" ; example boost
 
 ; Should we cache channel results on the Channels/Home screen?
-cache_home_channels = true
+cache_home_channels = false
```

This accidental change was previously reverted, but it seems to have come back again.

```suggestion
cache_home_channels = true
```